kernel/environ.ml

```diff
@@ -490,7 +490,10 @@ let lookup_projection cst env =
   Cmap_env.find (Projection.constant cst) env.env_globals.env_projections
 
 let is_projection cst env =
-  (lookup_constant cst env).const_proj
+  try
+    let _ = Cmap_env.find cst env.env_globals.env_projections in
```

Comment:
Once we change the representation of projections to include the inductive how will this work?

Reply:
I've got a wip PR on this, the goal is to move this function out of the kernel. (It's already not used there actually.)